syntax = "proto3";

package protocol;

message CleanUpMessage {
  string client_id = 1;
}
